### DISCUSSION TITLE
Command Line Issues with LaTeX Extensions

### DISCUSSION THREAD

**Discussion by wj-qiao** [sumatrapdfreader/sumatrapdf#4026] [General]
Hi there,

I'm using the latest version of SumatraPDF (v3.5.2) as my pdf viewer generated from VSCode Latex files, sometimes the reverse search command may fail. The following is the command line I used in the SumatraPDF:

"D:\Microsoft VS Code\Code.exe" "D:\Microsoft VS Code\resources\app\out\cli.js" --ms-enable-electron-run-as-node -r -g "%f:%l"

When it fails, I change to use the following command line:

"D:\Microsoft VS Code\Code.exe"  -g "%f":"%l"

The above two commands cannot guarantee effectiveness all the time, but there is always one that can be used. This confuses me, why can't we set it once and for all?

Thanks.

**Comment by GitHubRulesOK** [comment]
You should have set it once for whichever is the correct one (check with the workshop instructions) 
but there should not be a `-forward-search` signal after it has been added first time to settings, unless you are intentionally needing one!
